Src/AmrCore/AMReX_MFInterpolater.cpp

```diff
@@ -373,7 +377,9 @@ MFCellConsLinMinmaxLimitInterp::interp (MultiFab const& crsemf, int ccomp, Multi
             }
         });
 
-        Gpu::streamSynchronize();
```

Comment:
I think we need to keep this one because there is a local MulitiFab in this block.